libs/block-template-custom-element/src/dev.tsx

```diff
@@ -2,26 +2,11 @@ import { MockBlockDock } from "mock-block-dock";
 import { createRoot } from "react-dom/client";
 
 import packageJson from "../package.json";
+import { blockEntity, exampleGraph } from "./example-graph";
```

Comment:
I'm wondering if this should instead be importing from the generated file, which would catch more problems than importing directly from the generation script. For example, at the moment we do not use `blockEntityRecordId` in `exampleGraph` at all – it could become disconnected from `blockEntity` and nothing would break here. We should probably take `blockEntityRecordId` from `exampleGraph` at least, rather than `blockEntity.metadata.recordId`, if not directly consuming the generated file and just looking for the block entity in the entities.

Reply:
> should probably take `blockEntityRecordId` from `exampleGraph` at least, rather than `blockEntity.metadata.recordId`

We 100% should, I only added `blockEntityRecordId` as a thing after writing this and this was an oversight:
[Use exampleGraph.blockEntityRecordId](https://github.com/blockprotocol/blockprotocol/pull/1223/commits/1b0bd218c6865c512cf844653b7c798cb56d4e54)

> and nothing would break here 

With this change I think MBD should break, as it won't be able to find the entity you're identifying with the recordId